osu.Game/Screens/OnlinePlay/Matchmaking/MatchmakingCloud.cs

```diff
@@ -73,6 +82,13 @@ public MovingAvatar(APIUser apiUser)
                 Origin = Anchor.Centre;
             }
 
+            [BackgroundDependencyLoader]
+            private void load(AudioManager audio, SessionStatics statics)
+            {
+                playerAppearSample = audio.Samples.Get(@"UI/toolbar-hover");
+                lastSamplePlaybackTime = statics.GetBindable<double?>(Static.LastMatchmakingCloudSamplePlaybackTime);
```

Comment:
Does it need to be inside SessionStatics? May be easier to keep this as a local variable similar to `BeatmapSelectionOverlay`.